app/views/user_mailer/admin_deleted_work_notification.text.erb

```diff
@@ -1,11 +1,11 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
+<%= t 'user_mailer.admin_deleted_work_notification.hello', user: @user.default_pseud.byline %>
 
-Your work "<%= @work.title.html_safe %>" was deleted from the Archive by a site admin.
+<%= t('user_mailer.admin_deleted_work_notification.text.part1', title: @work.title)%> 
 
-If your work was part of an import project managed by our Open Doors team, please contact Open Doors (<%= "http://opendoors.transformativeworks.org/contact/open%20doors" %>) with any further questions.
+<%= t('user_mailer.admin_deleted_work_notification.text.part2', opendoors_link: 'http://opendoors.transformativeworks.org/contact/open%20doors')%>
 
-If it's possible your work violated the Archive's Terms of Service, please contact our Abuse Committee (<%= root_url + "abuse_reports/new" %>).
+<%= t('user_mailer.admin_deleted_work_notification.text.part3', abuse_link: root_url + "abuse_reports/new")%>
```

Comment:
This is missing a space between the closing `)` and the `%`. 

Would `new_abuse_report_url` work here instead of combining `root_url` and a string? If it works, it's simpler.